src/app/frontend/replicasetdetail/replicasetdetail_controller.js

```diff
@@ -30,6 +30,9 @@ export default class ReplicaSetDetailController {
     /** @export {!backendApi.Events} */
     this.replicaSetEvents = replicaSetEvents;
 
+    /** @export !Array<!backendApi.Event> */
+    this.events = replicaSetEvents.events;
+
     /** @const @export {!Array<string>} */
     this.eventTypeFilter = ['All', 'Warning'];
```

Comment:
Move 'All' and 'warning' to constants in this file? I see that you're using the values in a few places.